cli/src/diff_util.rs

```diff
@@ -211,7 +223,7 @@ impl BuiltinFormatKind {
         }
     }
 
-    fn to_arg_name(self) -> &'static str {
+    pub fn to_arg_name(self) -> &'static str {
```

Comment:
nit: maybe better to add a list of names (or `(name, variant)` pairs) instead of `ALL_VARIANTS`?

An "arg" name might be different from `:<name>`. If we add builtin format that doesn't support `--<name>` argument, `to_arg_name()` will return `Option<&'static str>`.

Reply:
> If we add builtin format that doesn't support --<name> argument, to_arg_name() will return Option<&'static str>.

I'm not sure I follow; I imagined all builtin formats would start with `:`, similarly to `:builtin`, `:ours`, etc.

But I don't mind making a list of all names. I think I'll then add an `BuiltinFormatKind::all_names()` function. I just learned that you can actually generate the list automatically in a const context, but it looks a little ugly.